vars/signArtifacts.groovy

```diff
@@ -6,13 +6,23 @@
  * compatible open source license.
  */
 
+/*
+SignArtifacts signs the given artifacts and saves the signature in the same directory
+@param Map[artifactPath] <Required> - Path to yml or artifact file.
+@param Map[component] <Optional> - Path to yml or artifact file.
+@param Map[type] <Optional> - Artifact type in the manifest, [type] is required for signing yml.
+@param Map[signatureType] <Optional> - The signature type of signing artifacts. e.g. '.sig'. Required for non-yml artifacts signing.
+@param Map[distributionPlatform] <Required> - The distribution platform for signing. Choices: ['linux']
```

Comment:
Let's remove `Choices: ['linux']` from here since this is not being tested in code.